.gitignore

```diff
@@ -15,7 +15,6 @@ dist/
 NOTICE.xml
 *.tgz
 *.tar.gz
-release-notes.md
```

Comment:
this file was removed a while ago

Reply:
IMO: We want to keep ignoring this so CI doesn't have to think about it when it's generated

Reply:
Oh! I didn't know it was generated! I just saw the one that was years old and assumed it's something we no longer did. I'll put it back in